CONTRIBUTING.md

```diff
@@ -2,6 +2,9 @@
 
 Thank you for your interest in contributing to the .NET documentation!
 
+> We're in the process of moving our guidelines into a site-wide contribution guide. 
```

Comment:
should't be better to use `>>>` instead of `>` for each line?

Reply:
How would that help? I think that triple blockquote looks worse and doesn't make sense.

Compare:

>>> We're in the process of moving our guidelines into a site-wide contribution guide. 

With:

> We're in the process of moving our guidelines into a site-wide contribution guide.

Reply:
This is my bad. I was confusing the markdown operator > here on GitHub with the >>> from Slack when we have multiples lines.